impl/sql/mutationstorage/mutations_test.go

```diff
@@ -150,119 +133,72 @@ func TestReadRange(t *testing.T) {
 		endSequence   uint64
 		count         int32
 		maxSequence   uint64
-		mutations     []*tpb.Entry
+		mutations     []*pb.EntryUpdate
 	}{
 		{
-			"read a single mutation",
-			0,
-			1,
-			1,
-			1,
-			[]*tpb.Entry{
-				{
-					Index:      []byte("index1"),
-					Commitment: []byte("mutation1"),
-				},
-			},
+			description:   "read a single mutation",
+			startSequence: 0,
+			endSequence:   1,
+			count:         1,
+			maxSequence:   1,
+			mutations:     []*pb.EntryUpdate{genUpdate(1)},
 		},
 		{
-			"empty mutations list",
-			100,
-			110,
-			10,
-			0,
-			nil,
+			description:   "empty mutations list",
+			startSequence: 100,
+			endSequence:   110,
+			count:         10,
+			maxSequence:   0,
+			mutations:     nil,
```

Comment:
It's generally fine to omit test struct fields that have the nil value for their type.
That would apply to mutations and maxSequence here, and startSequence in the case above.

Reply:
done